addons/popochiu/editor/popups/create_prop/create_prop.gd

```diff
@@ -75,7 +75,7 @@ func _create() -> void:
 	prop.clickable = _interaction_checkbox.button_pressed
 	prop.cursor = Constants.CURSOR_TYPE.ACTIVE
 	
-	if _new_prop_name in ['Bg', 'Background']:
+	if _new_prop_name in ['bg', 'background']:
```

Comment:
Wait! Are we going to change PascalCase in node names?

Reply:
Nope. Node names should be in PascalCase. The point of doing this here is that the name of the prop is changed to snake case when devs click the confirmation button in the popup. I can change this comparison to the `_pascal_name` variable if we want to keep consistency with Popochiu 1.x.